content/en/tutorials/upgrading_eks.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Upgrading Kubernetes Version on AWS"
+linkTitle: "Upgrading Kubernetes Version on AWS"
+date: 2022-01-03
+draft: false
+description: >
+  How to upgrade the version of your EKS cluster created by opta.
+---
+
+# Overview
+As Kubernetes is constantly [releasing new versions](https://kubernetes.io/releases/), and
+deprecating older versions, so to are the cloud provider implementations like EKS which Opta
+uses for Kubernetes on AWS. Unlike in GKE, in EKS these upgrades are not handled automatically
+and the users are expected to 
+[manually update their cluster themselves](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html).
+Fortunately for Opta users, this process does not need to be difficult and we have written an abridged version
+which should cover Opta-managed EKS setups.
+
+# Steps
+As Kubernetes always [maintains 1 minor version compatibility backwards or forwards](https://kubernetes.io/releases/version-skew-policy/),
+all Kubernetes upgrades should be done one minor version at a time. EKS actually enforces this process. This
+means that if, for example, one wishes to go from version 1.18 to 1.21, they will need to do the update of
+1.18 -> 1.19, then 1.19 -> 1.20 and finally 1.20 -> 1.21. You can often do these repetitions in quick succession,
+but please read the [Breaking Changes](#breaking-changes) section carefully as some upgrades may require
+extra steps.
+
+To begin the upgrade, go to your AWS console UI, to the EKS section, and to the details of the cluster in question
```

Comment:
We should somewhere mention that it can also be done through command line and post a link to AWS instructions. for users who don't want to use UI for the infra.